noxfile.py

```diff
@@ -188,7 +188,7 @@ def dev_test_sim(
         coverage_file.unlink()
 
     session.log(f"Running 'make test' against a simulator {config_str}")
-    session.run("make", "test", external=True, env=env)
+    session.run("make", "-k", "test", external=True, env=env)
```

Comment:
```suggestion
    session.run("make", "--keep-going", "test", external=True, env=env)
```
maybe? We require GNU make in the docs, so it should be available everywhere.